bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -265,7 +265,7 @@ spec:
               - args:
                 - --metrics-addr=127.0.0.1:8080
                 - --enable-leader-election
-                image: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:v0.39.0
+                image: ghcr.io/yurioliveirasa/opentelemetry-operator/opentelemetry-operator:v0.40.0
```

Comment:
This `yurioliveirasa` should be `open-telemetry` you should set this in `Makefile` before executing the `bundle`

Reply:
Fixed.